plugins/inputs/icecast/README.md

```diff
@@ -0,0 +1,60 @@
+# Icecast Input Plugin
+
+The icecast plugin gathers metrics from the Icecast listmount page enabling to see a detailed report of total listeners
+
+### Configuration:
+
+```toml
+# Read listeners from an Icecast instance per mount
+[[inputs.icecast]]
+  ## Specify the IP adress to where the '/admin/listmounts' can be found. You can include port if needed.
+  ## If you'd like to report under an alias, use ; (e.g. https://localhost;Server 1)
+  ## You can use multiple hosts who use the same login credentials by dividing with , (e.g. "http://localhost","https://127.0.0.1")
+  urls = ["http://localhost"]
+
+  ## Timeout to the complete conection and reponse time in seconds. Default (5 seconds)
+  # response_timeout = "25s"
+
+  ## The username/password combination needed to read the listmounts page.
+  ## These must be equal to the admin login details specified in your Icecast configuration
+  username = "admin"
+  password = "hackme"
+
+  ## Include the slash in mountpoint names or not
+  slash = false
```

Comment:
I think this option does not make sense here. If you don't like the slash, you can still replace it using a processor (e.g. starlark). Can you please elaborate on why you need it.